lib/player.rb

```diff
@@ -1,9 +1,15 @@
+
 class Player 
-  attr_reader :piece
+  attr_reader :piece, :name
   attr_accessor :winner
```

Comment:
I think we should make this a method instead maybe - a lot of our lessons have been hammering on accessors being pretty dangerous. I'll include that in my refactor of the player class later since we didn't initially scope it out in this set of work you tackled.

Reply:
Sounds good to me!